docs/source/topics/processes/usage.rst

```diff
@@ -728,9 +727,7 @@ The 'scheduled' indicates that the actual killing might not necessarily have hap
 This means that even after having called ``verdi process kill`` and getting the success message, the corresponding process may still be listed as active in the output of ``verdi process list``.
 
 By default, the ``pause``, ``play`` and ``kill`` commands will only ask for the confirmation of the runner that the request has been scheduled and not actually wait for the command to have been executed.
-This is because, as explained, the actual action being performed might not be instantaneous as the runner may be busy working with other processes, which would mean that the command would block for a long time.
-If you want to send multiple requests to a lot of processes in one go, this would be ineffective, as each one would have to wait for the previous one to be completed.
-To change the default and actually wait for the action to be completed and await its response, you can use the ``--wait`` flag.
+To change this behavior, you can use the ``--wait`` flag to actually wait for the action to be completed.
```

Comment:
Maybe as a compromise, just add this one sentence? If you are okay with this suggestion @danielhollas  I will merge the PR. I would like to close this one for the upcoming patch release.
```suggestion
To change this behavior, you can use the ``--wait`` flag to actually wait for the action to be completed.
If workers are under heavy load, it may take some time for them to respond to the request and for the command to finish.
```